Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.yml

```diff
@@ -0,0 +1,454 @@
+category: Endpoint
+commonfields:
+  id: Malwarebytes
+  version: -1
+configuration:
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Account ID
+  name: accountid
+  required: true
+  type: 0
+- display: Client ID
+  name: clientid
+  required: true
+  type: 0
+- display: Client Secret
+  name: clientsecret
+  required: true
+  type: 4
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: "Fetch Event List"
```

Comment:
why are the fetch related params single-select? what if i want to fetch threats of different categories with different severities? i think we should make them multi-select

Reply:
Because those features belong to different products. RTP belongs to EP and Suspicious activity belongs to EPR. Different categories of threats only apply to RTP events and Severity only applies to suspicious activities. there are no RTP events with severity and no suspicious activity events with malware category as it's not a scan detection. This was discussed before to implement this way.

Reply:
and what if i want to fetch suspicious activity with severities High and Medium?

Reply:
guess I can add an option "medium and above" for that case.

Reply:
would be easier to just make both params, category and severity, to be multi-select

Reply:
ok. i'll look into it

Reply:
if they choose, high and medium, what should be the severity level for the demisto incident ticket ?

Reply:
for now setting everything to high severity tickets. changed params to multiselect